VirtusLab-Open-Source/strapi-plugin-navigation
# fix: patch for version 2.2

## Ticket

https://github.com/VirtusLab/strapi-plugin-navigation/issues/257
https://github.com/VirtusLab/strapi-plugin-navigation/issues/258
https://github.com/VirtusLab/strapi-plugin-navigation/issues/259


## Summary

Fixes for version 2.2
- Drag and drop not working on firefox. Fixed by introducing a custom drag button component.
- Audience disappearing from navigation items after saving. Fixed audience parsing on request.
- Slug not showing in the configuration on the Settings Page. Fixed by adding 'UID' type to configuration options. 

## Test Plan

How are you testing the work you're submitting?

- Open the admin panel on the newest Firefox. It should be possible to reorder items by dragging them.  
- Add some audience fields to the element. Try collapsing the item and saving the navigation. The audience should not disappear from the navigation item. 
- Add an UID field to the content type and try configuring it to be a default title or default path on the Settings Page.
